text/0119-instance-options.md

```diff
@@ -0,0 +1,454 @@
+**CLI option**: `--dumps-dir`
+**Default value**: `dumps/`
+**Expected value**: a filepath pointing to a valid directory
+
+Sets the directory where Meilisearch will create dump files.
+
+### Import dump
+
+**Environment variable**: N/A
+**CLI option**: `--import-dump`
+**Default value**: `none`
+**Expected value**: a filepath pointing to a `.dump` file
+
+Imports the dump file located at the specified path. Path must point to a `.dump` file.
+
+Meilisearch will only launch once the dump data has been fully indexed.
+
+#### Log level
+
+**Environment variable**: `MEILI_LOG_LEVEL`
+**CLI option**: `--log-level`
+**Default value**: `'INFO'`
+**Expected value**: one of `ERROR`, `WARN`, `INFO`, `DEBUG`, OR `TRACE`
+
+Defines how much detail should be present in Meilisearch's logs.
+
+#### Max index size
+
+**Environment variable**: `MEILI_MAX_INDEX_SIZE`
+**CLI option**: `--max-index-size`
+**Default value**: `107374182400` (100 GiB)
+**Expected value**: an integer (`104857600`) or a human readable size (`100Mb`)
+
+Sets the maximum size of the index. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
+
+The `index` stores processed data and is different from the `task` database, which handles pending tasks.
+
+#### Max TASK_DB size
+
+**Environment variable**: `MEILI_MAX_TASK_DB_SIZE`
+**CLI option**: `--max-task-db-size`
+**Default value**: `107374182400` (100 GiB)
+**Expected value**: an integer (`104857600`) or a human readable size (`100Mb`)
+
+Sets the maximum size of the `task` database. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
+
+The `task` database handles pending tasks. This is different from the `index` database, which only stores processed data.
+
+#### Payload limit size
+
+**Environment variable**: `MEILI_HTTP_PAYLOAD_SIZE_LIMIT`
+**CLI option**: `--http-payload-size-limit`
+**Default value**: `104857600` (~100MB)
+**Expected value**: an integer (`104857600`) or a human readable size (`100Mb`)
+
+Sets the maximum size of accepted payloads. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
+
+#### Schedule snapshot creation
+
+**Environment variable**: `MEILI_SCHEDULE_SNAPSHOT`
+**CLI option**: `--schedule-snapshot`
+**Default**: Disabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Activates scheduled snapshots when enabled. Snapshots are disabled by default.
+
+#### Snapshot destination
+
+**Environment variable**: `MEILI_SNAPSHOT_DIR`
+**CLI option**: `--snapshot-dir`
+**Default value**: `snapshots/`
+**Expected value**: a filepath pointing to a valid directory
+
+Sets the directory where Meilisearch will store snapshots.
```

Comment:
```suggestion
Sets the directory where Meilisearch will store snapshots. If the directory does not exist when a snapshot is generated it will be created.
```